src/deploy/mod.rs

```diff
@@ -5,20 +5,22 @@ use crate::commands::subdomain::Subdomain;
 use crate::http;
 use crate::settings::global_user::GlobalUser;
 use crate::settings::toml::{DeployConfig, Zoneless};
-use crate::terminal::message::{Message, StdOut};
-pub fn worker(user: &GlobalUser, deploy_config: &DeployConfig) -> Result<(), failure::Error> {
+
+pub fn worker(
+    user: &GlobalUser,
+    deploy_config: &DeployConfig,
+) -> Result<(String, Vec<String>), failure::Error> {
```

Comment:
let's just return the list of URLs here; the message can be derived from them at the command level.